### DISCUSSION TITLE
HTTPS communication

### DISCUSSION THREAD

**Discussion by bozyman3000** [strimzi/strimzi-kafka-bridge#706] [Q&A]
Hello everyone, is there a plan to implement HTTPS communication for kafka-bridge in near future ? Thank you in advance.

**Comment by scholzj** [comment]
I do not think it will happen anytime soon - so no for near future. But it might happen long term.

In the meantime, depending on your requirements, you can for example consider:
* Fronting it with some proxy / API gateway which would provide the encryption between the client and the proxy / gateway
* Using load-balancers / Ingress / Routes etc. with TLS termination to provide some encryption support.
